stdlib/public/core/Result.swift

```diff
@@ -131,7 +131,7 @@ public enum Result<Success, Failure: Error> {
   ///     do {
   ///         let value = try integerResult.get()
   ///         print("The value is \(value).")
-  ///     } catch error {
+  ///     } catch let error {
```

Comment:
Good catch. I think the example can be simplified though: there's no need to bind the error for the didactic purposes here.

```suggestion
  ///     } catch {
```

Reply:
done